daemon/logger/splunk/splunk.go

```diff
@@ -228,12 +231,23 @@ func New(info logger.Info) (logger.Logger, error) {
 		}
 	}
 
+	timeout := defaultHTTPTimeout
+	if timeoutStr, ok := info.Config[splunkHTTPTimeout]; ok {
+		value, err := strconv.ParseInt(timeoutStr, 10, 64)
+
+		if err != nil {
+			return nil, err
+		}
+		timeout = time.Duration(value) * time.Millisecond
+	}
+
 	transport := &http.Transport{
 		TLSClientConfig: tlsConfig,
 		Proxy:           http.ProxyFromEnvironment,
 	}
 	client := &http.Client{
 		Transport: transport,
+		Timeout:   timeout,
```

Comment:
Perhaps (for now) we should not set a timeout by default, and keep the old behaviour, but allow setting a default (through the option that's added). We can alway decide to set one by default; wdyt?

Reply:
sounds great!